### PR TITLE
Make recording files visible in the specific section Media tab of the…

### LINKED ISSUE #8663: Make recording files visible in the specific section Media tab of the sidebar
Original issue: https://github.com/nextcloud/spreed/issues/8326

Original requirement:

> The recording shall be shared in the chat as a "normal file" but have a special section in the Media tab of the sidebar.

### PR DESCRIPTION
**Pull request by vitormattos**
### 🚧 TODO

- close #8663

### 🏁 Checklist

- [x] ⛑️ Tests (unit and/or integration) are included
- [x] 📘 API documentation in `docs/` has been updated or is not required
- [x] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required
- [x] 🔖 Capability is added or not needed

### CODE REVIEW

File: lib/Model/Attachment.php [hunk 1]
```diff
@@ -47,6 +47,8 @@ class Attachment extends Entity {
 	public const TYPE_MEDIA = 'media';
 	public const TYPE_OTHER = 'other';
 	public const TYPE_POLL = 'poll';
+	public const TYPE_RECORD_AUDIO = 'record-audio';
```
Comment (nickvergessen) [inline lib/Model/Attachment.php:50]: Needs to be documented in https://github.com/nextcloud/spreed/blob/062ed28540c21d09a19cf7077033e1b0165658c4/docs/constants.md#shared-item-types

**Review by nickvergessen — CHANGES_REQUESTED** [review]

**Comment by nickvergessen** [comment]
I'm actually unsure if it should be 2 sections or one, but then again with the different types it might be better in split sections?
cc @marcoambrosini

**Comment by nickvergessen** [comment]
PS screenshot in https://github.com/nextcloud/spreed/pull/8800

**Comment by vitormattos** [comment]
Updated the web client implementation here: #8800 
I unified the sections only on the client side.

**Comment by nickvergessen** [comment]
> I unified the sections only on the client side.

That's not good enough, as it needs more coordination with mobile clients, breaks the pagination, etc.

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**